cime_config/machines/config_machines.xml

```diff
@@ -3601,27 +3709,6 @@
       <env name="NUM_RS">$SHELL{echo "2*((`./xmlquery --value TOTAL_TASKS` + `./xmlquery --value TASKS_PER_NODE` - 1)/`./xmlquery --value TASKS_PER_NODE`)"|bc}</env>
       <env name="SMT_MODE">$SHELL{echo "(`./xmlquery --value MAX_TASKS_PER_NODE`+41)/42"|bc}</env>
     </environment_variables>
-    <environment_variables compiler="ibmgpu">
-      <env name="RS_PER_NODE">6</env>
-      <env name="CPU_PER_RS">7</env>
-      <env name="GPU_PER_RS">1</env>
-      <env name="LTC_PRT">gpu-cpu</env>
-      <env name="NUM_RS">$SHELL{echo "6*((`./xmlquery --value TOTAL_TASKS` + `./xmlquery --value TASKS_PER_NODE` - 1)/`./xmlquery --value TASKS_PER_NODE`)"|bc}</env>
-    </environment_variables>
-    <environment_variables compiler="pgigpu">
-      <env name="RS_PER_NODE">6</env>
-      <env name="CPU_PER_RS">3</env>
-      <env name="GPU_PER_RS">1</env>
-      <env name="LTC_PRT">gpu-cpu</env>
-      <env name="NUM_RS">$SHELL{echo "6*((`./xmlquery --value TOTAL_TASKS` + `./xmlquery --value TASKS_PER_NODE` - 1)/`./xmlquery --value TASKS_PER_NODE`)"|bc}</env>
-    </environment_variables>
-    <environment_variables compiler="gnugpu">
-      <env name="RS_PER_NODE">$SHELL{if [  `./xmlquery --value TOTAL_TASKS` -le 6 ];then echo  `./xmlquery --value  TOTAL_TASKS`; else echo 6; fi} </env>
-      <env name="CPU_PER_RS">7</env>
-      <env name="GPU_PER_RS">1</env>
-      <env name="LTC_PRT">gpu-cpu</env>
-	  <env name="NUM_RS">$SHELL{ if [ `./xmlquery --value TOTAL_TASKS` -le 6 ];then echo `./xmlquery --value TOTAL_TASKS`; else echo "6*((`./xmlquery --value TOTAL_TASKS` + `./xmlquery --value TASKS_PER_NODE` - 1)/`./xmlquery --value TASKS_PER_NODE`)"|bc; fi}</env>
```

Comment:
@xyuan You removed my changes to RS_PER_NODE and NUM_RS in your PR. This is needed when running on GPUs at small task count (<6) for debugging.